user_sync/connector/directory_ldap.py

```diff
@@ -197,6 +232,32 @@ def find_ldap_group_dn(self, group):
                 group_dn = current_tuple[0]
         return group_dn
 
+    def find_group_member_dn(self, group, member_attribute):
```

Comment:
should be plural: find_group_member_dns, but since it's an iterator it should be iter_group_member_dns